src/applications/vaos/appointment-list/pages/UpcomingAppointmentsDetailsPage/index.unit.spec.jsx

```diff
@@ -54,8 +54,8 @@ describe('VAOS Page: ConfirmedAppointmentDetailsPage with VAOS service', () => {
       });
 
       mockFacilityApi({
-        id: 983,
-        response: new MockFacilityResponse({ id: 983 }),
+        id: '983',
```

Comment:
Hmm would it be possible to extract the id from the response object directly? I think it's unnecessary to specify `983` as the id twice for `mockFacilityApi` but this isn't blocking.

Reply:
Yes, working to make work for either id or response.